examples/ui/core_widgets.rs

```diff
@@ -0,0 +1,233 @@
+//! This example illustrates how to create widgets using the `bevy_core_widgets` widget set.
+
+use bevy::{
+    color::palettes::basic::*,
+    core_widgets::{CoreButton, CoreWidgetsPlugin},
+    input_focus::{
+        tab_navigation::{TabGroup, TabIndex},
+        InputDispatchPlugin,
+    },
+    picking::hover::IsHovered,
+    prelude::*,
+    ui::{Depressed, InteractionDisabled},
+    winit::WinitSettings,
+};
+use bevy_ecs::system::SystemId;
+
+fn main() {
+    App::new()
+        .add_plugins((DefaultPlugins, CoreWidgetsPlugin, InputDispatchPlugin))
+        // Only run the app when there is user input. This will significantly reduce CPU/GPU use.
+        .insert_resource(WinitSettings::desktop_app())
+        .add_systems(Startup, setup)
+        .add_systems(
+            Update,
+            (update_button_style, update_button_style2, toggle_disabled),
+        )
+        .run();
+}
+
+const NORMAL_BUTTON: Color = Color::srgb(0.15, 0.15, 0.15);
+const HOVERED_BUTTON: Color = Color::srgb(0.25, 0.25, 0.25);
+const PRESSED_BUTTON: Color = Color::srgb(0.35, 0.75, 0.35);
+
+/// Marker which identifies buttons with a particular style, in this case the "Demo style".
+#[derive(Component)]
+struct DemoButton;
+
+fn update_button_style(
+    mut buttons: Query<
+        (
+            Has<Depressed>,
+            &IsHovered,
+            Has<InteractionDisabled>,
+            &mut BackgroundColor,
+            &mut BorderColor,
+            &Children,
+        ),
+        (
+            Or<(
+                Changed<Depressed>,
```

Comment:
Changed no longer makes sense as Depressed is a marker. This should be Added (like InteractionDisabled)